modules/genomic_browser/test/genomic_browserTest.php

```diff
@@ -163,58 +163,58 @@ function testGenomicBrowserFilterEachTab()
         $this->markTestSkipped("Skipping long test");
         return;
```

Comment:
Could also just remove the `return;` as code past `markTestSkipped()` is not executed.
Well, maybe Travis would flag it anyway.